components/layouts/QiskitOrgMenu.vue

```diff
@@ -1,353 +1,143 @@
-      top: 1px;
-      border-bottom: 4px solid $focus;
-    }
-  }
-
-  &--with-separator::before {
-    content: "";
-    background-color: $ui-03;
-    width: 2px;
-    margin: 12px 10px;
-  }
-
-  &--right-aligned {
-    margin-left: auto;
-  }
-
-  &--fixed {
-    /* fixings for mimicing qiskit.org menu */
-    margin-right: -0.4rem;
-  }
 
   @include mq($until: large) {
     display: none;
   }
 }
 
-.link-to-home {
-  @include type-style('productive-heading-02');
-  display: inline-flex;
-  align-items: center;
-  margin-left: -1.2rem;
-  padding: 0 1em;
-  color: var(--link-color);
-  text-decoration: none;
+.menu {
+  &__main-level {
+    @include framed();
+    padding-left: 0;
+    padding-right: 0;
+    border-bottom: 1px solid black;
+    justify-content: space-between;
 
-  @include mq($until: large) {
-    margin-left: -0.5rem;
+    &_on-left, &_on-right {
+      display: flex;
+    }
   }
-}
-
-.drawer {
-  display: none;
 
-  @include mq($until: large) {
-    display: unset;
-    margin-left: -2rem;
+  &__second-level {
+    justify-content: flex-end;
+    --link-color: #{$inverse-01};
+    background-color: $purple-40;
+    /* The menu should be framed() (max-width of 1100px = 68.75rem)
+    *  If we framed this menu, the purple backgroud fit the framed size, not all
+    *  That's why we calculate the padding, to align the secondary menu with the main one
+    */
+    padding-right: calc((100vw - 68.75rem) / 2);
+
+    &_on-right {
+      display: flex;
+    }
   }
-}
 
-.community-menu {
-  @include mq($until: large) {
+  &__mobile {
     display: none;
-  }
-}
 
-.drawer-toggle {
-  fill: white;
-  height: 100%;
-  cursor: pointer;
-  margin: 0 0 0 1.5rem;
-}
-
-.overlay {
-  position: fixed;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-  z-index: 150;
-  background-color: $ui-background;
-  opacity: 0;
-  transition: opacity 200ms;
-  pointer-events: none;
-}
-
-.vertical-navigation {
-  display: flex;
-  flex-direction: column;
-  position: fixed;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  z-index: 200;
-  width: 256px;
-  padding: 1.3rem;
-  background-color: $ui-background;
-  transform: translateX(-100%);
-  transition: transform 200ms;
-  overflow-y: auto;
-
-  h2 {
-    @include type-style('productive-heading-03');
-    color: $text-01;
-    padding: 1em;
-  }
-
-  &__item {
-    @include vertical-navigation-item();
-  }
-}
-
-.vertical-community-navigation {
-  display: flex;
-  flex-direction: column;
-  margin: 0 -1.3rem;
-  padding: 1rem 0;
-  background-color: $purple-30;
+    @include mq($until: large) {
+      display: flex;
+      height: 3.75rem;
+      cursor: pointer;
+      fill: white;
+    }
 
-  &__item {
-    @include vertical-navigation-item();
-    padding: 0.5rem 3rem;
-    color: $inverse-01;
+    &:focus {
+      .menu__overlay {
+        opacity: 0.5;
+      }
 
-    &.nuxt-link-active {
-      border-left: 4px solid $focus;
-      padding-left: calc(3rem - 4px);
+      .sidebar-menu {
```

Comment:
This component should style only `menu`-block-related elements. What we can do is to name the `<SidebarMenu>` component with a class such as `menu__side-menu` and stylish that.